### PR TITLE
Add tests to project

### PR DESCRIPTION
**Pull request by defint**

### CODE REVIEW

File: lib/loaders/e2e_file_translation_loader.dart [hunk 1]
```diff
@@ -2,24 +2,25 @@ import 'dart:async';
...
 
 class _CustomAssetBundle extends PlatformAssetBundle {
   Future<String> loadString(String key, {bool cache = true}) async {
-    final ByteData data = await load(key);
-    if (data == null) throw FlutterError('Unable to load asset: $key');
-
-    return utf8.decode(data.buffer.asUint8List());
+    try {
+      final ByteData data = await load(key);
+      return utf8.decode(data.buffer.asUint8List());
```
Comment (ilteoood) [inline lib/loaders/e2e_file_translation_loader.dart]: can you use a single point of return?

File: lib/loaders/e2e_file_translation_loader.dart [hunk 2]
```diff
@@ -2,24 +2,25 @@ import 'dart:async';
...
+    } catch (e) {
+      return "{}";
```
Comment (ilteoood) [inline lib/loaders/e2e_file_translation_loader.dart]: use this as default value, and log this error

File: lib/loaders/e2e_file_translation_loader.dart (hunk 1 above)
Comment (defint) [reply lib/loaders/e2e_file_translation_loader.dart]: Fixed

File: lib/loaders/e2e_file_translation_loader.dart (hunk 2 above)
Comment (defint) [reply lib/loaders/e2e_file_translation_loader.dart]: Did rollback as in base class: PlatformAssetBundle

**Pull request merged**